docs/FAQ.md

```diff
@@ -0,0 +1,36 @@
+# Frequently Asked Questions (FAQ)
+
+
+## APNs
+
+When iOS notifications do not work, double-check APNS configuration.
+Mainly the host, the key, and the bundle.
+
+APNs expose two hosts, production `api.push.apple.com` and development `api.sandbox.push.apple.com`.
+
+The typical misconfiguration is the usage of the dev key or the dev bundle with the production host or vice versa.
```

Comment:
Like the idea is good, but it should be done differently.

There should be errors from the connection to APNS and required action on our side - this is to generic.

Reply:
@kober32 @petrdvorak Are we really able to distinguish this kind of error?

Reply:
https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/handling_notification_responses_from_apns

Most typical cases:

BadDeviceToken - problem with the environment or with a token.

DeviceTokenNotForTopic - problem with bundle Id or topic (they do not match)

Reply:
Any idea how to rephrase it? My point was, that quite often, customers use the dev key against the production host.